src/components/Card.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { toast, ToastPosition, ToastType } from 'react-toastify';
+
+import { ReactComponent as CheckIcon } from 'assets/icons/check.svg';
+import { ReactComponent as MascotIcon } from 'assets/icons/mascot.svg';
+import { ReactComponent as TrashIcon } from 'assets/icons/trash.svg';
+import { Button, BUTTON } from 'components';
+import { UserActivity } from 'modules/activities';
+
+interface Props {
+  mode: 'list' | 'grid';
+  activity: UserActivity;
+  onRemove?: (id: string) => void;
+  onComplete?: (id: string) => void;
+}
+
+const CARD_STYLE = [
+  'card__deco--secondary',
+  'card__deco--tertiary',
+  'card__deco--quaternary',
+];
+
+export const Card: React.FC<Props> = ({
+  mode,
+  activity,
+  onRemove,
+  onComplete,
+}) => {
+  const [isLoading, setIsLoading] = React.useState(false);
+  const { title, score, style, id } = activity;
+  const isList = mode === 'list';
+  const cardClassName = isList ? 'card--list' : 'cardGrid';
+  const cardContentClassName = isList
```

Comment:
If you want to avoid a single if block here, then it would be ok to do this

```
const getCardClass = (selector: string) => isList ? `card__${selector}--list` : `card__${selector}--grid`;
const cardContentClassName = getCardClass('content');
const cardTextClassName = getCardClass('text');
...
```